backend/mastery/models.py

```diff
@@ -93,6 +93,10 @@ def get_affiliated_user(self, role_name):
             )
         return User.objects.filter(user_schools__school=self)
 
+    def get_userschool_users(self):
+        """Get all users directly affiliated with this school via UserSchool."""
+        return User.objects.filter(user_schools__school=self).distinct()
+
```

Comment:
My main problem with this function isn't what it does, but that its name doesn't clearly fall in line with the similar functions on our models.

But do we need this at all? Is there a case where you can't get the same done using `get_affiliated_users`? If there is, maybe we can build upon that function instead?